lib/archive/zip/extra_field/unix.rb

```diff
@@ -124,16 +124,6 @@ def dump_central
     #
     # Returns a String suitable to writing to a local file record in a ZIP
     # archive file which contains the data for this object.
-    def dump_local
-      [
-        ID,
-        12 + @data.size,
-        @atime.to_i,
-        @mtime.to_i,
-        @uid,
-        @gid
-      ].pack('vvVVvv') + @data
-    end
     alias :dump_local :dump_central
```

Comment:
Actually, I believe this alias needs to be reversed and the dump_central method removed instead.